static/lang/re-en.json

```diff
@@ -1797,6 +1797,9 @@
                 "TargetHuntedPrey": "Target is your Hunted Prey",
                 "TrackHuntedPrey": "To Track hunted prey"
             },
+            "HyldarfsFang": {
+                "Note": "Fire damage dealt by this weapon (including persistent fire damage and damage from Hyldarf's Venom) ignores the target's fire resistance."
```

Comment:
This is near identical to the greater flaming rune wording so we can just use that instead.

Reply:
hmm, that has a note about the critical persistent damage. That shouldn't be needed in that note anyways.